Client/src/Views/ProfileView.tsx

```diff
@@ -0,0 +1,153 @@
+import TableCell from '@material-ui/core/TableCell'
+import TableContainer from '@material-ui/core/TableContainer'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Typography from '@material-ui/core/Typography'
+import { exampleDatasets } from '../Models/Datasets/ICompleteDatasetEntity'
+import { makeStyles } from '@material-ui/core/styles'
+
+const renderGraphRow = (row) => {
+  return (<Table size="small" aria-label="purchases">
+    <TableHead>
+      <TableRow>
+        <TableCell>Graph image</TableCell>
+        <TableCell>Datasets in graph</TableCell>
+        <TableCell>Comments</TableCell>
+      </TableRow>
+    </TableHead>
+    <TableBody>
+      <TableRow >
+        <TableCell component="th" scope="row">
+          <img src={DataboomTestGraph} width="200" height="200" />
+        </TableCell>
+        <TableCell>{row.graphdatasets}</TableCell>
+        <TableCell>{row.comments}</TableCell>
+      </TableRow>
+    </TableBody>
+  </Table>)
+}
+
+const useRowStyles = makeStyles({
+  root: {
+    '& > *': {
+      borderBottom: 'unset',
+    },
+  },
+})
+
+//no oxidizer, add material detials and composition, comments keep author
+function createData(id: number[], type: string, name: string, oxidizer: string, author: string, graphdatasets: string[], comments: string) {
+  return { id, type, name, oxidizer, author, graphdatasets, comments }
+}
+
+const rows = []
+
+for (let i = 0; i < exampleDatasets.length; i++) {
+  rows.push(createData([exampleDatasets[i].id], 'Dataset', exampleDatasets[i].name, exampleDatasets[i].oxidizer, exampleDatasets[i].author,
+    ['Cell width', 'Cell Height of O2 explosion', 'Critical energy after N2 intake'], 'Here is a sample comment'))
+}
+
+function createGraphData(id: number[], type: string, name: string, graphdatasets: string[], comments: string) {
+  return { id, type, name, graphdatasets, comments }
+}
+
+rows.push(createGraphData([1, 2, 3], 'Graph', 'graph name', ['Cell width', 'Cell Height of O2 explosion', 'Critical energy after N2 intake'],
+  'Here is a sample comment'))
+
+function parseIdArray(id) {
+  let stringTest = ""
+  console.log(id.length)
+  if (id.length > 1) {
+    stringTest = String(id).split(",").join("&datasetId[]=")
+    return stringTest
+  } else { return id }
+}
+
+// eslint-disable-next-line no-undef
+function Row(props: { row: ReturnType<typeof createData> }) {
+  const { row } = props
+  const [open, setOpen] = React.useState(false)
+  const classes = useRowStyles()
+  return (
+    < React.Fragment >
+      <TableRow className={classes.root}>
+        <TableCell>
+          <IconButton aria-label="expand row" size="small" onClick={() => setOpen(!open)}>
+            {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+          </IconButton>
+        </TableCell>
+        <TableCell component="th" scope="row">
+          <Link to={"graph?datasetId[]=" + String(parseIdArray(row.id))}>{row.name}</Link>
```

Comment:
why not just datasetIds=

Reply:
That was a request from Sean that he wants the string to be in this format